Geometry_Engine/Modify/Extend.cs

```diff
@@ -21,22 +21,208 @@
  */
 
 using BH.oM.Geometry;
+using BH.oM.Geometry.CoordinateSystem;
+using BH.oM.Reflection.Attributes;
+using System;
+using System.Collections.Generic;
 using System.Linq;
 
 namespace BH.Engine.Geometry
 {
     public static partial class Modify
     {
         /***************************************************/
-        /**** Public Methods - Curves                   ****/
+        /**** Private Methods - Curves                  ****/
         /***************************************************/
 
-        public static Line Extend(this Line curve, double start = 0.0, double end = 0.0)
+        private static Line Extend(this Line curve, double start = 0.0, double end = 0.0)
```

Comment:
any reason this method was made private?

Reply:
> any reason this method was made private?

![image](https://user-images.githubusercontent.com/26874773/65945920-485d8980-e435-11e9-89f8-d977673a047f.png)

Reply:
But this specific method was already in the code and has been there for over 2 years. See no reason why this should be made private now?

Reply:
@pawelbaran not this time actually, that one already was in the code and I mindlessly made it private while replacing every other `public` with `private`

Reply:
Yes, @IsakNaslundBh, you're right. I'll fix it

Reply:
Ah, right - my bad!